src/generate-common-seo.js

```diff
@@ -48,10 +48,13 @@ export function generateStructuredData(config = {}) {
   const { "theme-attributes":themeConfig, "social-links":socialLinks, "seo-metadata":seoMetadata = [] } = config;
   const homePageSeo = seoMetadata.find(page => page["owner-type"] === "home") || {};
   const { "page-title":pageTitle = "", description = "", keywords  =  "" } = get(homePageSeo, ["data"], {});
-
   if(!themeConfig || !themeConfig.logo) {
     return {};
   }
+  let enableStructuredDataForNewsArticle = themeConfig['structured_data_news_article'] || false;
+  if(config.hasOwnProperty('enableStructuredDataForNewsArticle') && typeof config.enableStructuredDataForNewsArticle !== "undefined"){
```

Comment:
`config.hasOwnProperty('enableStructuredDataForNewsArticle') && config.enableStructuredDataForNewsArticle ` should work

Reply:
we want to check **config.enableStructuredDataForNewsArticle !== "undefined"** because **config.enableStructuredDataForNewsArticle** can also be false

Reply:
as per your condition only if it's true it will execute the if block